interfaces/functions.yaml

```diff
@@ -2080,6 +2080,11 @@ igraph_induced_subgraph_map:
         OPTIONAL OUT INDEX_VECTOR map, OPTIONAL OUT INDEX_VECTOR invmap
     DEPS: vids ON graph
 
+# default product type=Cartesian
```

Comment:
```suggestion
```

Let's remove the redundant comment.

Using `CARTESIAN` below is good. This value will be associated with the enum value on the C side in each high-level interface that makes use of the interface definition file.